Source/WebKit/Shared/RemoteLayerTree/RemoteLayerBackingStoreCollection.mm

```diff
@@ -63,6 +63,15 @@
         backingStore->paintContents();
 }
 
+bool RemoteLayerBackingStoreCollection::hasAnyReachableBackingStoreWithNonEmptyDirtyRegion() const
+{
+    for (auto* backingStore : m_backingStoresNeedingDisplay) {
+        if (!backingStore->hasEmptyDirtyRegion())
+            return true;
+    }
+    return false;
+}
```

Comment:
Can't we remove this function and make it the output of `RemoteLayerBackingStoreCollection::paintReachableBackingStoreContents()` since these two functions are called together. So `paintReachableBackingStoreContents()` will paint and will return true if it has already painted something.